xds/internal/balancer/clusterresolver/resource_resolver_eds.go

```diff
@@ -92,13 +93,39 @@ func (er *edsDiscoveryMechanism) OnError(err error) {
 		return
 	}
 
-	er.topLevelResolver.onError(err)
+	if er.logger.V(2) {
+		er.logger.Infof("EDS discovery mechanism for resource %q reported error: %v", er.nameToWatch, err)
+	}
+
+	// If a previously received good configuration exists, continue to use it.
+	// Else report an empty update that would result in the child policy
```

Comment:
"child priority of priority corresponding to EDS." also drop the no priorities or localities, there would still be a hierarchy of priorities, but one of them will report TF.

Reply:
As mentioned in the other review comment, an empty update would result in that child not being created. I updated the comment anyways, making it reflect reality more than it was previously doing.

Reply:
Oh ok I see, I just remember having to add TF back from weighted target if addr list is zero in UpdateClientConnState. That's prob why I was under the impression the system was two children with top priority child and second priority child, and first sends TF back. The first one just not being created makes sense too haha :).